docs/intro/contributing.txt

```diff
@@ -117,38 +117,41 @@ Download the Django source code repository using the following command:
 
 Now that you have a local copy of Django, you can install it just like you would
 install any package using ``pip``. The most convenient way to do so is by using
-a *virtual environment* (or virtualenv) which is a feature built into Python
+a *virtual environment* which is a feature built into Python
 that allows you to keep a separate directory of installed packages for each of
 your projects so that they don't interfere with each other.
 
-It's a good idea to keep all your virtualenvs in one place, for example in
-``.virtualenvs/`` in your home directory. Create it if it doesn't exist yet:
+It's a good idea to keep all your virtual environments in one place, for example
+in ``.virtualenvs/`` in your home directory.
 
-.. console::
-
-    $ mkdir ~/.virtualenvs
-
-Now create a new virtualenv by running:
+Create a new virtual environment by running:
 
 .. console::
 
-    $ python -m venv ~/.virtualenvs/djangodev
+    $ python3 -m venv ~/.virtualenvs/djangodev
 
 The path is where the new environment will be saved on your computer.
 
-.. admonition:: For Ubuntu users
+.. admonition:: For Debian/Ubuntu systems
 
-    On some versions of Ubuntu the above command might fail. Use the
-    ``virtualenv`` package instead, first making sure you have ``pip3``:
+    On Debian/Ubuntu systems the above command will fail. It will tell
```

Comment:
I think 'might fail' is still valid since Python might have been installed from somewhere other than the package manager

Reply:
`On unmodified Debian/Ubuntu systems...`

Reply:
the original wording: `might fail` isn't quite right either because this python3-venv error message is expected to appear in most cases for people that havn't already fixed it